saleor/graphql/attribute/mutations/attribute_value_create.py

```diff
@@ -63,23 +65,34 @@ def clean_input(cls, info: ResolveInfo, instance, data, **kwargs):
         input_type = instance.attribute.input_type
 
         is_swatch_attr = input_type == AttributeInputType.SWATCH
-        errors: dict[str, list[ValidationError]] = {}
+        errors: dict[str, list[ValidationError]] = defaultdict(list)
         if not is_swatch_attr:
             for field in cls.ONLY_SWATCH_FIELDS:
                 if cleaned_input.get(field):
-                    errors[field] = [
+                    errors[field].append(
                         ValidationError(
                             f"The field {field} can be defined only for swatch attributes.",  # noqa: E501
                             code=AttributeErrorCode.INVALID.value,
                         )
-                    ]
+                    )
         else:
             try:
                 cls.validate_swatch_attr_value(cleaned_input)
             except ValidationError as error:
                 errors["value"] = error.error_dict[cls.ATTRIBUTE_VALUES_FIELD]
                 errors["fileUrl"] = error.error_dict[cls.ATTRIBUTE_VALUES_FIELD]
 
+        if input_type == AttributeInputType.NUMERIC and "name" in cleaned_input:
+            try:
+                float(cleaned_input["name"])
+            except ValueError:
+                errors["name"].append(
+                    ValidationError(
+                        "Value of numeric attribute must be numeric.",
+                        code=AttributeErrorCode.INVALID.value,
+                    )
+                )
```

Comment:
It shouldn't be here. This mutation is only for creating the values for attributes with choices. The `numeric` attribute is not this kind of attribute. If it's possible, the validation of that is missing, not for validating the numeric value.

Reply:
Thanks Iga. I can change the validation, but I would like to know:

1. Where did you get this info, that the mutation is for attribute with choices only?
2. What other mutation can be used, to create numeric attribute value?

I need to know that in order to display a proper error description.

Reply:
1. Unfortunately it's not described anyway directly, but the input for this mutation has a deprecation message for some fields that suggested this: https://github.com/saleor/saleor/blob/main/saleor/graphql/attribute/mutations/attribute_create.py#L32
Furthermore, attributes without choices are created per instance, so you need to specify instance when creating it, this mutation doesn't take such value, so you will end up with `AttributeValue` that is a trash.
2. You need to use the instance mutation for that, like `ProductUpdate`, `ProductVariantUpdate`, `ProductVariantCreate`. It depends on to which object you want to create an attribute value